tests/target/normalize_multiline_doc_attribute.rs

```diff
@@ -0,0 +1,7 @@
+// rustfmt-unstable: true
+// rustfmt-normalize_doc_attributes: true
+
+///This comment
```

Comment:
I think that we should add a leading space on each line.

Reply:
There was a [comment](https://github.com/rust-lang/rustfmt/pull/3548/files#diff-4b9f3caeb7d15ba0422250fb31e55930L333) about this in the original code. Maybe I should add it again?

Reply:
Oh sorry. Please keep the current implementation, it is the correct behavior (cc https://doc.rust-lang.org/rustdoc/the-doc-attribute.html).